### ISSUE TITLE
weighted cross entropy loss function

### ISSUE THREAD

**Issue by kamil-kaczmarek** [minerva-ml/open-solution-mapping-challenge#15]
Build training that implements this:
* erode masks (so all masks were separated, they didn’t touch),
* train U-Net with weighted cross entropy loss function. Idea described in this [kaggle post](https://www.kaggle.com/c/data-science-bowl-2018/discussion/54426). While training add more weight in loss function to points that we would assign to *contours touching* category,
* make predictions,
* dilate predictions.

Weights calculated as in the [U-Net paper](https://arxiv.org/pdf/1505.04597.pdf) (page 5, eq. 2).
